docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.        distance : float
</pre><p>@NimaSarajpoor</p><p>change the name to <code>nn_distance</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Will take care of it.</p>